src/game/board.ts

```diff
@@ -114,6 +114,26 @@ export const getNextLocation = (
   return null;
 };
 
+export const getDistance = (
+  source: number,
+  destination: number,
+  direction: Direction
+): number | number | null => {
```

Comment:
I can't immediately see how it's happening, but something about this breaks if you select the very first square, which is location 0 (doesn't show the play highlight).

It looks like we're checking everything versus null here, instead of `!source` or whatever, so maybe it's in Square.tsx

Reply:
It was Square.tsx.

Only other thing here is, this could use some tests!  🤓

Reply:
LOL check out your lazy board tests
I can definitely do those